### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

**Comment by afollestad** [comment]
@naman14 Travis reports:
```
/home/travis/build/naman14/TimberX/app/src/main/res/values/secret.xml:10: Error: "cast_app_id" is not translated in "nl" (Dutch) [MissingTranslation]
      <string name="cast_app_id">123456</string>
```
I think the the mock value needs `translatable="false"`

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <string name="sort_order_entry_year">Jaar</string>
+    <string name="sort_order_entry_duration">Duur</string>
+
+    <string name="bullet">\u2022</string>
+
+    <string name="number_song_add_playlist"><xliff:g example="1" id="number">%d</xliff:g> nummer toegevoegd aan afspeellijst.</string>
+    <string name="number_songs_add_playlist"><xliff:g example="1" id="number">%d</xliff:g> nummers toegevoegd aan afspeellijst.</string>
+
+    <!--do not translate these strings-->
+    <string name="durationformatshort" translatable="false"><xliff:g id="format">%2$d:%3$02d</xliff:g></string>
```
Comment (afollestad) [inline app/src/main/res/values-nl/strings.xml]: These non-translatable strings can be totally left out of this file :)

**Comment by afollestad** [comment]
@Vistaus you may want to look at the strings in the app once more - I've made more parts of the app translatable (see strings.xml and strings-about.xml)

**Comment by Vistaus** [comment]
@afollestad Will do :-)

**Comment by Vistaus** [comment]
Translated the newly added strings as well, see #21 Will close this PR.

**Pull request closed without merging**